src/test/Manual.test.js

```diff
@@ -0,0 +1,148 @@
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000001,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000010,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000100,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00001000,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00010000,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00100000,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 01000000,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 10000000,00000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000001,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000010,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000100,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00001000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00010000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00100000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,01000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,10000000,00000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000001,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000010,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000100,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00001000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00010000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00100000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,01000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,10000000,00000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,00000001',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,00000010',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,00000100',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,00001000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,00010000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,00100000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,01000000',
+    'Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 00000000,00000000,00000000,10000000',
+];
+
+
+test('Sends correct Actuator command',async () => {
+    const { getByText } = render(Manual,{test_ui: true})
+    // Set output to watch
+    const output = getByText('Test: 0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0')
+    //Loop through all actuators
+    for (let index = 0; index < 36; index++) {
+        const button = getByText('Button ' + index.toString())
+        await fireEvent.click(button)
+        expect(output).toHaveTextContent(groundTruth[index])
+    }
+
+})
```

Comment:
Nice. The passing of props can be very extendable from here. If we start really building up a lot of tests, this pattern will create problems with our deployment bundle size. We have a loooong way to go before we might hit that though.